src/base/Context.ts

```diff
@@ -205,7 +205,15 @@ export class Context {
    * @note the signing Account will be set to the Signing Manager's first Account. If the Signing Manager has
    *   no Accounts yet, the signing Account will be left empty
    */
-  public async setSigningManager(signingManager: SigningManager): Promise<void> {
+  public async setSigningManager(signingManager: SigningManager | null): Promise<void> {
+    if (signingManager === null) {
+      this._signingManager = undefined;
+      this.signingAddress = undefined;
+      // note polkadot API does not currently support unsetting a signer
```

Comment:
I looked at the upstream code, and we could safely do `this_.polymeshApi.setSigner(undefined as any)`, however we would be giving up type safety with that.

We could submit a PR upstream allowing this, but we'd need to provide reasoning and need to wait for its release + updating our code to use latest.

I think this is OK, although exposed publicly as an escape hatch the polkadotAPI is meant to be mainly an internal API.